### ISSUE TITLE
Build broken for so long?

### ISSUE THREAD

**Issue by keilw** [unitsofmeasurement/seshat#17]
Why is the build broken for such a long time, @desruisseaux any idea or something you changed?

**Comment by desruisseaux** [comment]
I was not aware that it was broken. Thanks for spotting it! [The CI said](https://app.circleci.com/pipelines/github/unitsofmeasurement/seshat):

> No `.circleci/config.yml` was found in your project.

But [that file exists](https://github.com/unitsofmeasurement/seshat/blob/master/.circleci/config.yml). I will investigate more tomorrow. I'm very novice in CircleCI, so if you have any hints, than would be greatly appreciated!

**Comment by keilw** [comment]
That's weird, but can you check the version in that YML, it might be outdated and no longer recognized especially if it's 1.x. You can see the files in the API or other project repos as a guideline.

**Comment by desruisseaux** [comment]
I think the key error is:

```
Failed to collect dependencies at tech.units:unit-tck:jar:1.1 ->
Failed to read artifact descriptor for org.jboss.test-audit:jboss-test-audit-impl:jar:1.1.0.Final:
Could not transfer artifact org.jboss.test-audit:jboss-test-audit-impl:pom:1.1.0.Final from/to
maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories:
    [jcenter (http://jcenter.bintray.com, default, releases+snapshots),
    bintray-release (http://oss.jfrog.org/artifactory/libs-release, default, releases),
    bintray-snapshot (http://oss.jfrog.org/artifactory/libs-snapshot, default, releases+snapshots),
    bintray-uom (http://dl.bintray.com/unitsofmeasurement/maven, default, releases+snapshots)]
```

So it may be because the jcenter/bintray repositories are now retired. I presume that those dependencies have been fixed in `tech.units:unit-tck:2.1.1` and that the best action would be to upgrade (at last) Apache SIS and Seshat to JSR-385. I agree that I'm very, very, very, sooo late on this long due work…

For short term I will check if I can override the repository link.

**Comment by desruisseaux** [comment]
Fixed as a side-effect of #14.
